### ISSUE TITLE
Test and Deploy on Heroku fails on npm install 

### ISSUE THREAD

**Issue by robotiko** [Strider-CD/strider#716]
Error trace: 
$ git checkout -qf master 
$ npm install 
node: /usr/lib/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by node)

Heroku issue? any way to solve this?

**Comment by gunish** [comment]
+1

**Comment by reactivepixel** [comment]
+1

The deploy to heroku button fails with "!     Push rejected, failed to compile Multipack app"

**Comment by frankcarey** [comment]
Here is the full output:

```

-----> Fetching custom git buildpack... done
-----> Multipack app detected
=====> Downloading Buildpack: https://github.com/mbuchetics/heroku-buildpack-nodejs-grunt
=====> Detected Framework: Node.js
       PRO TIP: Avoid using semver ranges starting with '>' in engines.node
       See https://devcenter.heroku.com/articles/nodejs-support
-----> Requested node range:  > 0.8.x
-----> Resolved node version: 0.12.3
-----> Downloading and installing node
-----> Writing a custom .npmrc to circumvent npm bugs
-----> Exporting config vars to environment
-----> Installing dependencies
       npm WARN deprecated swig@0.14.0: v1.0.0 is a complete rewrite of Swig from the ground up. Previous versions are no longer supported
       npm WARN engine global-tunnel@1.2.0: wanted: {"node":"0.10.x"} (current: {"node":"0.12.3","npm":"2.9.1"})
       npm WARN optional dep failed, continuing fsevents@0.3.0
       npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"0.12.3","npm":"2.9.1"})
       npm WARN engine npm@1.3.26: wanted: {"node":">=0.6","npm":"1"} (current: {"node":"0.12.3","npm":"2.9.1"})
       npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"0.12.3","npm":"2.9.1"})
       npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"0.12.3","npm":"2.9.1"})
       npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"0.12.3","npm":"2.9.1"})
       npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"0.12.3","npm":"2.9.1"})
       
       > ws@0.5.0 install /tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/socket.io/node_modules/engine.io/node_modules/ws
       > (node-gyp rebuild 2> builderror.log) || (exit 0)
       
       make: Entering directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/socket.io/node_modules/engine.io/node_modules/ws/build'
         CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
         SOLINK_MODULE(target) Release/obj.target/bufferutil.node
         SOLINK_MODULE(target) Release/obj.target/bufferutil.node: Finished
         COPY Release/bufferutil.node
         CXX(target) Release/obj.target/validation/src/validation.o
         SOLINK_MODULE(target) Release/obj.target/validation.node
         SOLINK_MODULE(target) Release/obj.target/validation.node: Finished
         COPY Release/validation.node
       make: Leaving directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/socket.io/node_modules/engine.io/node_modules/ws/build'
       npm WARN installMany nopt was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       npm WARN installMany npmlog was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       npm WARN installMany request was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       npm WARN installMany semver was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       npm WARN installMany tar was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       npm WARN installMany tar-pack was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       npm WARN installMany mkdirp was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       npm WARN installMany rc was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       npm WARN installMany rimraf was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
       
       > ws@0.4.31 install /tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws
       > (node-gyp rebuild 2> builderror.log) || (exit 0)
       
       make: Entering directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/build'
         CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
       make: Leaving directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/build'
       
       > kerberos@0.0.11 install /tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos
       > (node-gyp rebuild 2> builderror.log) || (exit 0)
       
       make: Entering directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos/build'
         CXX(target) Release/obj.target/kerberos/lib/kerberos.o
         CXX(target) Release/obj.target/kerberos/lib/worker.o
         CC(target) Release/obj.target/kerberos/lib/kerberosgss.o
         CC(target) Release/obj.target/kerberos/lib/base64.o
         CXX(target) Release/obj.target/kerberos/lib/kerberos_context.o
         SOLINK_MODULE(target) Release/obj.target/kerberos.node
         SOLINK_MODULE(target) Release/obj.target/kerberos.node: Finished
         COPY Release/kerberos.node
       make: Leaving directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos/build'
       npm WARN package.json github-url-from-git@1.1.1 No repository field.
       npm WARN installMany normalize-package-data was bundled with npm@1.3.26, but bundled package wasn't found in unpacked tree
       
       > kerberos@0.0.9 install /tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos
       > (node-gyp rebuild 2> builderror.log) || (exit 0)
       
       make: Entering directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos/build'
         CXX(target) Release/obj.target/kerberos/lib/kerberos.o
         CXX(target) Release/obj.target/kerberos/lib/worker.o
         CC(target) Release/obj.target/kerberos/lib/kerberosgss.o
         CC(target) Release/obj.target/kerberos/lib/base64.o
         CXX(target) Release/obj.target/kerberos/lib/kerberos_context.o
         SOLINK_MODULE(target) Release/obj.target/kerberos.node
         SOLINK_MODULE(target) Release/obj.target/kerberos.node: Finished
         COPY Release/kerberos.node
       make: Leaving directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos/build'
       
       > bson-ext@0.1.6 install /tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/bson/node_modules/bson-ext
       > ./node_modules/node-pre-gyp/bin/node-pre-gyp.js install --fallback-to-build
       
       make: Entering directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/bson/node_modules/bson-ext/build'
         CXX(target) Release/obj.target/bson/ext/bson.o
         SOLINK_MODULE(target) Release/obj.target/bson.node
         SOLINK_MODULE(target) Release/obj.target/bson.node: Finished
         COPY Release/bson.node
       make: Leaving directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/bson/node_modules/bson-ext/build'
       
       > bson@0.2.21 install /tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/mongoose/node_modules/mongodb/node_modules/bson
       > (node-gyp rebuild 2> builderror.log) || (exit 0)
       
       make: Entering directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/mongoose/node_modules/mongodb/node_modules/bson/build'
         CXX(target) Release/obj.target/bson/ext/bson.o
         SOLINK_MODULE(target) Release/obj.target/bson.node
         SOLINK_MODULE(target) Release/obj.target/bson.node: Finished
         COPY Release/bson.node
       make: Leaving directory `/tmp/build_70090f81420d97421df7ee9318d9f7d8/Strider-CD-strider-2095c7c/node_modules/mongoose/node_modules/mongodb/node_modules/bson/build'
       connect-slashes@0.0.11 node_modules/connect-slashes
       
       pw@0.0.4 node_modules/pw
       
       connect-flash@0.1.1 node_modules/connect-flash
       
       cookie@0.0.5 node_modules/cookie
       
       ansiparse@0.0.5-1 node_modules/ansiparse
       
       gravatar@1.1.1 node_modules/gravatar
       
       eventemitter2@0.4.14 node_modules/eventemitter2
       
       extend@2.0.1 node_modules/extend
       
       everypaas@0.0.7 node_modules/everypaas
       
       thirty-two@0.0.2 node_modules/thirty-two
       
       consolidate@0.10.0 node_modules/consolidate
       
       step@0.0.5 node_modules/step
       
       async@0.9.0 node_modules/async
       
       strider-build-badge@0.1.0 node_modules/strider-build-badge
       
       passport-local@1.0.0 node_modules/passport-local
       └── passport-strategy@1.0.0
       
       cookie-parser@1.3.4 node_modules/cookie-parser
       ├── cookie-signature@1.0.6
       └── cookie@0.1.2
       
       MD5@1.2.1 node_modules/MD5
       ├── charenc@0.0.1
       └── crypt@0.0.1
       
       path-to-regexp@1.1.1 node_modules/path-to-regexp
       └── isarray@0.0.1
       
       validator@3.39.0 node_modules/validator
       
       strider-custom@0.5.4 node_modules/strider-custom
       └── gitane@0.3.1
       
       ssh-keypair@1.1.0 node_modules/ssh-keypair
       └── crypto@0.0.3
       
       cors@2.6.0 node_modules/cors
       └── vary@1.0.0
       
       passport@0.2.1 node_modules/passport
       ├── passport-strategy@1.0.0
       └── pause@0.0.1
       
       rc@0.5.5 node_modules/rc
       ├── strip-json-comments@0.1.3
       ├── deep-extend@0.2.11
       ├── ini@1.3.3
       └── minimist@0.0.10
       
       semver@4.3.4 node_modules/semver
       
       method-override@2.3.3 node_modules/method-override
       ├── methods@1.1.1
       ├── vary@1.0.0
       ├── parseurl@1.3.0
       └── debug@2.2.0 (ms@0.7.1)
       
       bcryptjs@2.1.0 node_modules/bcryptjs
       
       morgan@1.5.3 node_modules/morgan
       ├── basic-auth@1.0.1
       ├── depd@1.0.1
       ├── on-finished@2.2.1 (ee-first@1.1.0)
       └── debug@2.2.0 (ms@0.7.1)
       
       serve-favicon@2.2.1 node_modules/serve-favicon
       ├── fresh@0.2.4
       ├── ms@0.7.1
       ├── parseurl@1.3.0
       └── etag@1.6.0 (crc@3.2.1)
       
       express-session@1.11.2 node_modules/express-session
       ├── cookie-signature@1.0.6
       ├── cookie@0.1.2
       ├── on-headers@1.0.0
       ├── utils-merge@1.0.0
       ├── parseurl@1.3.0
       ├── depd@1.0.1
       ├── crc@3.2.1
       ├── uid-safe@2.0.0 (base64-url@1.2.1)
       └── debug@2.2.0 (ms@0.7.1)
       
       cli-table@0.3.1 node_modules/cli-table
       └── colors@1.0.3
       
       mailer@0.6.7 node_modules/mailer
       ├── colors@1.1.0
       └── nodemailer@0.1.20 (mimelib-noiconv@0.1.9)
       
       strider-git@0.2.2 node_modules/strider-git
       ├── shell-escape@0.0.1
       ├── gitane@0.3.1
       └── fs-extra@0.8.1 (jsonfile@1.1.1, rimraf@2.2.8, ncp@0.4.2, mkdirp@0.3.5)
       
       swig@0.14.0 node_modules/swig
       └── underscore@1.8.3
       
       compression@1.4.4 node_modules/compression
       ├── vary@1.0.0
       ├── on-headers@1.0.0
       ├── bytes@1.0.0
       ├── debug@2.2.0 (ms@0.7.1)
       ├── compressible@2.0.2 (mime-db@1.9.1)
       └── accepts@1.2.7 (negotiator@0.5.3, mime-types@2.0.11)
       
       errorhandler@1.3.6 node_modules/errorhandler
       ├── escape-html@1.0.1
       └── accepts@1.2.7 (negotiator@0.5.3, mime-types@2.0.11)
       
       superagent@0.21.0 node_modules/superagent
       ├── extend@1.2.1
       ├── methods@1.0.1
       ├── cookiejar@2.0.1
       ├── component-emitter@1.1.2
       ├── reduce-component@1.0.1
       ├── qs@1.2.0
       ├── mime@1.2.11
       ├── debug@2.2.0 (ms@0.7.1)
       ├── readable-stream@1.0.27-1 (isarray@0.0.1, inherits@2.0.1, string_decoder@0.10.31, core-util-is@1.0.1)
       ├── form-data@0.1.3 (combined-stream@0.0.7)
       └── formidable@1.0.14
       
       bluebird@2.9.25 node_modules/bluebird
       
       winston@0.8.3 node_modules/winston
       ├── cycle@1.0.3
       ├── stack-trace@0.0.9
       ├── isstream@0.1.2
       ├── eyes@0.1.8
       ├── colors@0.6.2
       ├── async@0.2.10
       └── pkginfo@0.3.0
       
       express@4.12.3 node_modules/express
       ├── utils-merge@1.0.0
       ├── cookie-signature@1.0.6
       ├── methods@1.1.1
       ├── fresh@0.2.4
       ├── cookie@0.1.2
       ├── escape-html@1.0.1
       ├── merge-descriptors@1.0.0
       ├── content-type@1.0.1
       ├── vary@1.0.0
       ├── parseurl@1.3.0
       ├── range-parser@1.0.2
       ├── finalhandler@0.3.4
       ├── serve-static@1.9.2
       ├── content-disposition@0.5.0
       ├── path-to-regexp@0.1.3
       ├── depd@1.0.1
       ├── on-finished@2.2.1 (ee-first@1.1.0)
       ├── qs@2.4.1
       ├── debug@2.1.3 (ms@0.7.0)
       ├── etag@1.5.1 (crc@3.2.1)
       ├── proxy-addr@1.0.8 (forwarded@0.1.0, ipaddr.js@1.0.1)
       ├── send@0.12.2 (destroy@1.0.3, ms@0.7.0, mime@1.3.4)
       ├── type-is@1.6.2 (media-typer@0.3.0, mime-types@2.0.11)
       └── accepts@1.2.7 (negotiator@0.5.3, mime-types@2.0.11)
       
       body-parser@1.12.4 node_modules/body-parser
       ├── bytes@1.0.0
       ├── content-type@1.0.1
       ├── depd@1.0.1
       ├── on-finished@2.2.1 (ee-first@1.1.0)
       ├── qs@2.4.2
       ├── debug@2.2.0 (ms@0.7.1)
       ├── raw-body@2.0.1 (bytes@2.0.1)
       ├── type-is@1.6.2 (media-typer@0.3.0, mime-types@2.0.11)
       └── iconv-lite@0.4.8
       
       strider-heroku@0.1.1 node_modules/strider-heroku
       ├── ssh-keypair@1.0.1 (crypto@0.0.3)
       ├── strider-git@0.1.1 (shell-escape@0.0.1, gitane@0.3.1, mkdirp@0.3.5)
       ├── passport-heroku@0.0.2 (pkginfo@0.2.3, passport-oauth@0.1.15)
       └── superagent@0.15.7 (methods@0.0.1, debug@0.7.4, cookiejar@1.3.0, qs@0.6.5, emitter-component@1.0.0, reduce-component@1.0.1, mime@1.2.5, formidable@1.0.14)
       
       strider-ruby@0.0.2 node_modules/strider-ruby
       └── strider-detection-rules@0.0.1 (gumshoe@0.2.1)
       
       strider-python@0.2.1 node_modules/strider-python
       └── strider-detection-rules@0.0.1 (gumshoe@0.2.1)
       
       lodash@2.4.2 node_modules/lodash
       
       strider-slack@2.0.0 node_modules/strider-slack
       ├── ejs@1.0.0
       ├── lodash.foreach@2.4.1 (lodash.forown@2.4.1, lodash._basecreatecallback@2.4.1)
       └── slackihook@1.0.0 (needle@0.7.11)
       
       nodemailer@0.6.5 node_modules/nodemailer
       ├── public-address@0.1.1
       ├── directmail@0.1.8
       ├── he@0.3.6
       ├── readable-stream@1.1.13 (isarray@0.0.1, inherits@2.0.1, string_decoder@0.10.31, core-util-is@1.0.1)
       ├── simplesmtp@0.3.35 (xoauth2@0.1.8, rai@0.1.12)
       └── mailcomposer@0.2.12 (mime@1.2.11, dkim-signer@0.1.2, follow-redirects@0.0.3, mimelib@0.2.19)
       
       connect@2.29.1 node_modules/connect
       ├── utils-merge@1.0.0
       ├── cookie-signature@1.0.6
       ├── cookie@0.1.2
       ├── fresh@0.2.4
       ├── pause@0.0.1
       ├── content-type@1.0.1
       ├── on-headers@1.0.0
       ├── bytes@1.0.0
       ├── parseurl@1.3.0
       ├── response-time@2.3.1
       ├── vhost@3.0.0
       ├── basic-auth-connect@1.0.0
       ├── depd@1.0.1
       ├── qs@2.4.1
       ├── http-errors@1.3.1 (inherits@2.0.1, statuses@1.2.1)
       ├── debug@2.1.3 (ms@0.7.0)
       ├── connect-timeout@1.6.2 (ms@0.7.1, debug@2.2.0)
       ├── finalhandler@0.3.4 (escape-html@1.0.1, on-finished@2.2.1)
       ├── express-session@1.10.4 (crc@3.2.1, uid-safe@1.1.0)
       ├── csurf@1.7.0 (csrf@2.0.7)
       ├── type-is@1.6.2 (media-typer@0.3.0, mime-types@2.0.11)
       ├── serve-static@1.9.2 (escape-html@1.0.1, send@0.12.2)
       ├── multiparty@3.3.2 (stream-counter@0.2.0, readable-stream@1.1.13)
       └── serve-index@1.6.4 (escape-html@1.0.1, batch@0.5.2, accepts@1.2.7, debug@2.2.0, mime-types@2.0.11)
       
       request@2.55.0 node_modules/request
       ├── caseless@0.9.0
       ├── aws-sign2@0.5.0
       ├── stringstream@0.0.4
       ├── json-stringify-safe@5.0.0
       ├── forever-agent@0.6.1
       ├── form-data@0.2.0
       ├── oauth-sign@0.6.0
       ├── tunnel-agent@0.4.0
       ├── isstream@0.1.2
       ├── qs@2.4.2
       ├── node-uuid@1.4.3
       ├── combined-stream@0.0.7 (delayed-stream@0.0.5)
       ├── mime-types@2.0.11 (mime-db@1.9.1)
       ├── bl@0.9.4 (readable-stream@1.0.33)
       ├── http-signature@0.10.1 (assert-plus@0.1.5, asn1@0.1.11, ctype@0.5.3)
       ├── tough-cookie@1.1.0
       ├── har-validator@1.7.0 (commander@2.8.1, chalk@1.0.0, is-my-json-valid@2.11.0)
       └── hawk@2.3.1 (cryptiles@2.0.4, sntp@1.0.9, boom@2.7.1, hoek@2.13.0)
       
       httpcheck@0.0.1 node_modules/httpcheck
       └── request@2.16.6 (forever-agent@0.2.0, aws-sign@0.2.0, tunnel-agent@0.2.0, oauth-sign@0.2.0, cookie-jar@0.2.0, json-stringify-safe@3.0.0, mime@1.2.11, node-uuid@1.4.3, form-data@0.0.10, qs@0.5.6, hawk@0.10.2)
       
       strider-mailer@0.2.1 node_modules/strider-mailer
       ├── lodash.foreach@2.4.1 (lodash.forown@2.4.1, lodash._basecreatecallback@2.4.1)
       └── nodemailer@0.5.15 (public-address@0.1.1, directmail@0.1.8, he@0.3.6, readable-stream@1.0.33, simplesmtp@0.3.35, mailcomposer@0.2.12)
       
       strider-ssh-deploy@0.4.2 node_modules/strider-ssh-deploy
       ├── ejs@1.0.0
       ├── progress-stream@0.5.0 (single-line-log@0.3.1, speedometer@0.1.4, through2@0.2.3)
       ├── ssh2@0.3.6 (streamsearch@0.1.2, asn1@0.2.1, readable-stream@1.0.27-1)
       └── undefined (fstream-npm@0.1.8, tar@0.1.20)
       
       strider-bitbucket@1.0.3 node_modules/strider-bitbucket
       ├── debug@0.7.4
       ├── gravatar@1.0.6
       ├── crypto@0.0.3
       ├── async@0.2.10
       ├── passport-bitbucket@0.1.2 (passport-oauth@0.1.15, pkginfo@0.2.3)
       ├── oauth@0.9.12
       ├── passport@0.1.18 (pause@0.0.1, pkginfo@0.2.3)
       ├── superagent@0.15.7 (methods@0.0.1, cookiejar@1.3.0, qs@0.6.5, emitter-component@1.0.0, reduce-component@1.0.1, mime@1.2.5, formidable@1.0.14)
       ├── lodash@2.2.1
       └── request@2.27.0 (json-stringify-safe@5.0.0, aws-sign@0.3.0, forever-agent@0.5.2, qs@0.6.6, tunnel-agent@0.3.0, oauth-sign@0.3.0, cookie-jar@0.3.0, mime@1.2.11, node-uuid@1.4.3, form-data@0.1.4, http-signature@0.10.1, hawk@1.0.0)
       
       strider-gitlab@1.0.5 node_modules/strider-gitlab
       ├── async@0.2.10
       ├── strider-git@0.2.2 (shell-escape@0.0.1, gitane@0.3.1, fs-extra@0.8.1)
       ├── lodash@2.2.1
       └── request@2.27.0 (json-stringify-safe@5.0.0, aws-sign@0.3.0, forever-agent@0.5.2, qs@0.6.6, tunnel-agent@0.3.0, oauth-sign@0.3.0, cookie-jar@0.3.0, mime@1.2.11, node-uuid@1.4.3, form-data@0.1.4, http-signature@0.10.1, hawk@1.0.0)
       
       strider-github@1.3.0 node_modules/strider-github
       ├── ssh-keypair@1.0.1
       ├── debug@0.7.4
       ├── scmp@0.0.2
       ├── gravatar@1.0.6
       ├── crypto@0.0.3
       ├── async@0.2.10
       ├── passport@0.1.18 (pause@0.0.1, pkginfo@0.2.3)
       ├── passport-github@0.1.7 (passport-oauth2@1.1.2)
       ├── superagent@0.15.7 (methods@0.0.1, cookiejar@1.3.0, qs@0.6.5, emitter-component@1.0.0, reduce-component@1.0.1, mime@1.2.5, formidable@1.0.14)
       ├── lodash@2.2.1
       └── request@2.27.0 (json-stringify-safe@5.0.0, forever-agent@0.5.2, aws-sign@0.3.0, qs@0.6.6, tunnel-agent@0.3.0, oauth-sign@0.3.0, cookie-jar@0.3.0, mime@1.2.11, node-uuid@1.4.3, form-data@0.1.4, http-signature@0.10.1, hawk@1.0.0)
       
       global-tunnel@1.2.0 node_modules/global-tunnel
       ├── tunnel@0.0.2
       └── lodash@1.3.1
       
       strider-metadata@0.0.2 node_modules/strider-metadata
       └── lodash@1.3.1
       
       strider-simple-runner@0.13.1 node_modules/strider-simple-runner
       ├── async@0.1.22
       ├── dirkeeper@0.2.0 (rimraf@2.2.8, async@0.2.10)
       ├── fs-extra@0.8.1 (jsonfile@1.1.1, rimraf@2.2.8, ncp@0.4.2, mkdirp@0.3.5)
       ├── strider-runner-core@2.0.0 (shell-quote@1.3.3, async@0.2.10, mkdirp@0.3.5)
       └── lodash@1.3.1
       
       strider-env@0.4.4 node_modules/strider-env
       └── lodash@1.3.1
       
       jade@0.35.0 node_modules/jade
       ├── character-parser@1.2.0
       ├── commander@2.0.0
       ├── mkdirp@0.3.5
       ├── transformers@2.1.0 (promise@2.0.0, css@1.0.8, uglify-js@2.2.5)
       ├── with@1.1.1 (uglify-js@2.4.0)
       ├── constantinople@1.0.2 (uglify-js@2.4.21)
       └── monocle@1.1.50 (readdirp@0.2.5)
       
       strider-email-notifier@0.4.2 node_modules/strider-email-notifier
       ├── underscore@1.5.2
       ├── async@0.2.10
       └── moment@2.4.0
       
       strider-cli@1.4.8 node_modules/strider-cli
       ├── rimraf@2.3.3
       ├── debug@2.2.0 (ms@0.7.1)
       ├── chokidar@0.9.0 (recursive-readdir@0.0.2)
       ├── touch@0.0.3 (nopt@1.0.10)
       ├── nomnom@1.8.1 (underscore@1.6.0, chalk@0.4.0)
       ├── glob@4.5.3 (inherits@2.0.1, once@1.3.2, inflight@1.0.4, minimatch@2.0.7)
       ├── superagent@0.20.0 (extend@1.2.1, methods@1.0.1, cookiejar@2.0.1, component-emitter@1.1.2, reduce-component@1.0.1, qs@1.2.0, mime@1.2.11, debug@2.0.0, readable-stream@1.0.27-1, form-data@0.1.3, formidable@1.0.14)
       ├── prompt@0.2.14 (revalidator@0.1.8, pkginfo@0.3.0, read@1.0.5, utile@0.2.1)
       └── strider-ecosystem-client@1.2.1 (needle@0.7.11, js-yaml@3.3.1)
       
       moment@2.10.3 node_modules/moment
       
       socket.io@1.3.5 node_modules/socket.io
       ├── debug@2.1.0 (ms@0.6.2)
       ├── has-binary-data@0.1.3 (isarray@0.0.1)
       ├── socket.io-adapter@0.3.1 (object-keys@1.0.1, debug@1.0.2, socket.io-parser@2.2.2)
       ├── socket.io-parser@2.2.4 (isarray@0.0.1, debug@0.7.4, component-emitter@1.1.2, benchmark@1.0.0, json3@3.2.6)
       └── engine.io@1.5.1 (base64id@0.1.0, debug@1.0.3, engine.io-parser@1.2.1, ws@0.5.0)
       
       socket.io-client@1.3.5 node_modules/socket.io-client
       ├── to-array@0.1.3
       ├── debug@0.7.4
       ├── indexof@0.0.1
       ├── object-component@0.0.3
       ├── component-bind@1.0.0
       ├── component-emitter@1.1.2
       ├── backo2@1.0.2
       ├── has-binary@0.1.6 (isarray@0.0.1)
       ├── parseuri@0.0.2 (better-assert@1.0.2)
       ├── socket.io-parser@2.2.4 (isarray@0.0.1, benchmark@1.0.0, json3@3.2.6)
       └── engine.io-client@1.5.1 (component-inherit@0.0.3, xmlhttprequest@1.5.0, debug@1.0.4, parseqs@0.0.2, parsejson@0.0.1, parseuri@0.0.4, engine.io-parser@1.2.1, has-cors@1.0.3, ws@0.4.31)
       
       strider-extension-loader@0.4.5 node_modules/strider-extension-loader
       ├── methods@0.0.1
       ├── underscore@1.5.2
       ├── async@0.2.10
       ├── connect@2.4.6 (pause@0.0.1, fresh@0.1.0, bytes@0.1.0, cookie@0.0.4, crc@0.2.0, debug@2.2.0, send@0.0.4, formidable@1.0.11, qs@0.5.1)
       └── less@1.4.2 (mime@1.2.11, mkdirp@0.3.5, ycssmin@1.0.1)
       
       strider-node@0.3.5 node_modules/strider-node
       ├── n@1.1.0
       ├── async@0.2.10
       ├── MD5@1.1.0 (charenc@0.0.1, crypt@0.0.1)
       ├── fs-extra@0.8.1 (jsonfile@1.1.1, rimraf@2.2.8, ncp@0.4.2, mkdirp@0.3.5)
       ├── strider-detection-rules@0.0.1 (gumshoe@0.2.1)
       └── npm@1.3.26 (normalize-package-data@2.1.0)
       
       connect-mongo@0.8.1 node_modules/connect-mongo
       ├── depd@1.0.1
       ├── debug@2.2.0 (ms@0.7.1)
       └── mongodb@2.0.31 (readable-stream@1.0.31, mongodb-core@1.1.26)
       
       mongoose@3.8.28 node_modules/mongoose
       ├── regexp-clone@0.0.1
       ├── sliced@0.0.5
       ├── muri@1.1.0
       ├── hooks@0.2.1
       ├── mpath@0.1.1
       ├── mpromise@0.4.3
       ├── ms@0.1.0
       ├── mquery@1.4.0 (debug@0.7.4, bluebird@2.3.2)
       └── mongodb@1.4.32 (readable-stream@1.0.33, kerberos@0.0.9, bson@0.2.21)
-----> Caching node_modules directory for future builds
-----> Cleaning up node-gyp and npm artifacts
-----> Building runtime environment
=====> Downloading Buildpack: https://github.com/heroku/heroku-buildpack-ruby.git
 !     Push rejected, failed to compile Multipack app
```

**Comment by frankcarey** [comment]
Issue may be with the buildpack itself.. https://github.com/ddollar/heroku-buildpack-multi/pull/47

**Comment by knownasilya** [comment]
@frankcarey could you spearhead this and test out the fork?

**Comment by dancourse** [comment]
+1 same error

**Comment by olalonde** [comment]
+1 same error when deploying to Dokku https://github.com/progrium/dokku/issues/1244

**Comment by knownasilya** [comment]
Closing, since it was probably fixed with #818
